config.go

```diff
@@ -1022,6 +1024,46 @@ func ValidateConfig(cfg Config, usageMessage string,
 			numNets++
 			cfg.ActiveNetParams = chainreg.BitcoinSimNetParams
 		}
+		if cfg.Bitcoin.SigNet {
```

Comment:
Let's return an error if signet is configured for litecoin?

Reply:
Good idea, done.